qa/c8-orchestration-cluster-e2e-test-suite/utils/requestHelpers/incident-requestHelpers.ts

```diff
@@ -0,0 +1,134 @@
+                },
+            },
+        });
+
+        await assertStatusCode(res, 200);
+        await validateResponse(
+            {
+                path: INCIDENT_SEARCH_ENDPOINT,
+                method: 'POST',
+                status: '200',
+            },
+            res,
+        );
+        const json = await res.json();
+        expect(json.items.length).toBeGreaterThan(0);
+        localState['incidents'] = json.items;
+    }).toPass(defaultAssertionOptions);
+    
+    return localState['incidents'] as {
+        incidentKey: string,
+        errorMessage: string,
+        processDefinitionId: string,
+        processDefinitionKey: string,
+        processInstanceKey: string,
+        elementInstanceKey: string,
+        elementId: string,
+        creationTime: string,
+        tenantId: string,
+        jobKey?: string,
+    }[];
+}
+
+/**
+ * Create Instance of one process with two incidents of the same type
+ * @param localState 
+ * @param request 
+ */
+export async function setupIncidentTest (
+  localState: Record<string, unknown>,
+  request: APIRequestContext,
+) {
+    await test.step('Create process instance with an incident', async () => {
+        const instances = await createInstances('processWithAnError', 1, 1);
+        localState['processInstanceKey'] = instances[0].processInstanceKey;
+    });
+
+    await test.step('Search incident to get incidentKey', async () => {
+        const incidents = await searchIncidentByPIK(request, {
+          processInstanceKey: localState['processInstanceKey'] as string,
+        });
+        localState['incidentKeys'] = incidents.map(incident => incident.incidentKey);
+      });
+}
+
+/**
+ * Create two process instances with incidents of different type
+ */
+export async function setupMultipleIncidentsTest(
+  localState: Record<string, unknown>,
+  request: APIRequestContext,
+) {
+    await test.step('Create process instances with incidents', async () => {
+        const instances = [];
+        const processWithAnErrorInstance = await createSingleInstance('processWithAnError', 1,);
+        instances.push(processWithAnErrorInstance.processInstanceKey);
+
+        const loanApprovalProcessInstance = await createSingleInstance('loanApprovalProcess', 1,);
+        instances.push(loanApprovalProcessInstance.processInstanceKey);
+        localState['processInstanceKey'] = instances;
+    });
+
+    await test.step('Search incident to get incidentKey', async () => {
+        const allKeys: Array<number | string> = [];
+        for (const key of localState['processInstanceKey'] as string[]) {
+            const incidents = await searchIncidentByPIK(request, { processInstanceKey: key });
+            allKeys.push(...(incidents ?? []).map(i => i.incidentKey));
+        }
+
+        localState['incidentKeys'] = allKeys;
+      });
+}
+
+/**
+ * Create one process instance with two incidents of different type
+ */
+export async function setupMultipleTypeIncidentsTest(
```

Comment:
As this signature and `setupMultipleIncidentsTest` are pretty close together I would suggest to rename them to what they actually do like
`createIncidentsInTwoProcess`, `createTwoIncidentsInOneProcess`